new-website/src/components/location/index.jsx

```diff
@@ -61,6 +61,21 @@ export const LocationOverview = () => {
         })
     }
 
+    const onSuccess = (msg) => {
+        setAlert({
+            open: true,
+            message: formatMessage({ id: msg }),
+            severity: "success",
+        })
+    }
+
+    React.useEffect(() => {
```

Comment:
This use effect will run at every render. Please try to add a second parameter when this useEffect should run.
https://reactjs.org/docs/hooks-reference.html#conditionally-firing-an-effect

Reply:
Done. Used empty dependency array.